### PR TITLE
support new ca/sa binary annotations

### PR DESCRIPTION
**Pull request by bmdhacks**

### CODE REVIEW

File: zipkin-web/src/main/resources/public/js/zipkin-span.js [hunk 1]
```diff
@@ -97,6 +97,12 @@ Zipkin.fromRawSpan = function(rawSpan) {
   });
   var kvAnnotations = $.map(rawSpan.binaryAnnotations, function(b) {
     var ann =  Zipkin.fromRawKvAnnotation(b);
+    // convert ca/sa annotations to more readable
+    if (ann.key == "ca" || ann.key == "sa") {
+      ann.key = (ann.key == "ca") ? "Client Address" : "Server Address";
+      ann.setAnnotationType(6); // string
+      ann.setValue(ann.host.ipv4 + ":" + ann.host.port);
+    }
```
Comment (franklinhu) [inline zipkin-web/src/main/resources/public/js/zipkin-span.js:105]: Are the other core annotations converted on the server? Might make sense to do it all in the same place

File: zipkin-web/src/main/resources/public/js/zipkin-kv-annotation.js [hunk 2]
```diff
@@ -22,17 +22,27 @@
...
   KvAnnotation.prototype.getKey            = function() { return this.key; };
   KvAnnotation.prototype.getValue          = function() { return this.value; };
   KvAnnotation.prototype.getAnnotationType = function() { return this.annotationType; };
   KvAnnotation.prototype.getSpan           = function() { return this.span; };
+  KvAnnotation.prototype.getHost           = function() { return this.host; };
+  KvAnnotation.prototype.getHostAddress    = function() {
+    if (this.host !== undefined) {
+     return this.host.ipv4
+    } else {
+     return "0.0.0.0"
```
Comment (franklinhu) [inline zipkin-web/src/main/resources/public/js/zipkin-kv-annotation.js]: semicolons? :trollface:

File: zipkin-web/src/main/resources/public/js/zipkin-span.js (hunk 1 above)
Comment (bmdhacks) [inline zipkin-web/src/main/resources/public/js/zipkin-span.js:105]: good point.

Comment (bmdhacks) [inline zipkin-web/src/main/resources/public/js/zipkin-span.js:105]: No they're not

**Pull request closed without merging**